### PR TITLE
Redash

### PR DESCRIPTION
**Pull request by tony0306**
## Summary
Onboarding Redash. PostgreSQL and Redis are included by the redash community chart. 

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by michaeljguarino** [comment]
@tony0306 I think we should actually focus on lightdash not redash

**Comment by michaeljguarino** [comment]
We can still put this in though, there are people who do want it

**Comment by tony0306** [comment]
I think that I'm pretty close to finish it, once done I will move to lightdash as well. I guess next week this might be done as well.

File: redash/helm/redash/values.yaml.tpl [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  application:
+    links:
+    - description: redash web ui
+      url: {{ .Values.hostname }}
+
+redash:
+  redash:
+    secretKey: {{ dedupe . "redash.redash.redash.secretKey" (randAlphaNum 32) }}
+    cookieSecret: {{ dedupe . "redash.redash.redash.cookieSecret" (randAlphaNum 32) }}
+  postgresql:
```
Comment (michaeljguarino) [inline redash/helm/redash/values.yaml.tpl:12]: these values should be in the base values.yaml not here

File: redash/helm/redash/values.yaml.tpl [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    enabled: false
+  redis:
+    enabled: false
+  externalRedisSecret:
```
Comment (michaeljguarino) [inline redash/helm/redash/values.yaml.tpl:16]: similarly can go in plain values.yaml

File: redash/plural/notes.tpl [hunk 3]
```diff
@@ -0,0 +1 @@
+Use `plural watch redash` to track the status of your application
```
Comment (michaeljguarino) [inline redash/plural/notes.tpl:1]: we should add a line like:

`View your redash instance at {{ .Values.hostname }}`

**Review by michaeljguarino — APPROVED** [review]

**Pull request merged**